_includes/v2.0/orchestration/kubernetes-prometheus-alertmanager.md

```diff
@@ -0,0 +1,185 @@
+
+        <img src="{{ 'images/v2.1/kubernetes-prometheus-graph.png' | relative_url }}" alt="Prometheus graph" style="border:1px solid #eee;max-width:100%" />
+
+    {{site.data.alerts.callout_success}}
+    Prometheus auto-completes CockroachDB time series metrics for you, but if you want to see a full listing, with descriptions, port-forward as described in [Access the Admin UI](#access-the-admin-ui) and then point your browser to [http://localhost:8080/_status/vars](http://localhost:8080/_status/vars).
+
+    For more details on using the Prometheus UI, see their [official documentation](https://prometheus.io/docs/introduction/getting_started/).
+    {{site.data.alerts.end}}
+
+### Configure Alertmanager
+
+Active monitoring helps you spot problems early, but it is also essential to send notifications when there are events that require investigation or intervention. This section shows you how to use [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) and CockroachDB's starter [alerting rules](https://github.com/cockroachdb/cockroach/blob/master/cloud/kubernetes/prometheus/alert-rules.yaml) to do this.
+
+1. Download our <a href="https://raw.githubusercontent.com/cockroachdb/cockroach/master/cloud/kubernetes/prometheus/alertmanager-config.yaml" download><code>alertmanager-config.yaml</code></a> configuration file.
+
+2. Edit the `alertmanager-config.yaml` file to [specify the desired receivers for notifications](https://prometheus.io/docs/alerting/configuration/). Initially, the file contains a dummy web hook.
+
+3. Add this configuration to the Kubernetes cluster as a secret, renaming it to `alertmanager.yaml` and labelling it to make it easier to find:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ kubectl create secret generic alertmanager-cockroachdb --from-file=alertmanager.yaml=alertmanager-config.yaml
+    ~~~
+
+    ~~~
+    secret "alertmanager-cockroachdb" created
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ kubectl label secret alertmanager-cockroachdb app=cockroachdb
+    ~~~
+
+    ~~~
+    secret "alertmanager-cockroachdb" labeled
+    ~~~
+
+4. Use our [`alertmanager.yaml`](https://github.com/cockroachdb/cockroach/blob/master/cloud/kubernetes/prometheus/alertmanager.yaml) file to create the various objects necessary to run an Alertmanager instance, including a ClusterIP service so that Prometheus can forward alerts:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ kubectl apply -f https://raw.githubusercontent.com/cockroachdb/cockroach/master/cloud/kubernetes/prometheus/alertmanager.yaml
+    ~~~
+
+    ~~~
+    alertmanager "cockroachdb" created
+    service "alertmanager-cockroachdb" created
+    ~~~
+
+    {{site.data.alerts.callout_danger}}
+    Do not change the secret name or the contents of the `alertmanager.yaml` file. Doing so will XXXX
```

Comment:
@bobvawter, this is my only open question: What bad things happen if you change the secret name or contents of this file?